File: src/main/java/com/fish/service/AcceptedService.java
package com.fish.service;

import com.fish.common.Result;
import com.fish.entity.pojo.Accepted;
import com.mybatisflex.core.service.IService;

import java.util.ArrayList;

public interface AcceptedService extends IService<Accepted> {
    int addAccepted(Accepted accepted);
    Result<ArrayList<Accepted>> getAccepts();
    Result<ArrayList<Accepted>> getMyAccepts(String userId);
    Result<ArrayList<Accepted>> getTopicAccepts(Long topicId);
}

File: src/main/kotlin/com/fish/kexieOnlineJudge/service/topic/AcceptedService.kt
package com.fish.kexieOnlineJudge.service.topic

import com.fish.kexieOnlineJudge.entity.pojo.topic.Accepted
import com.mybatisflex.core.service.IService

interface AcceptedService : IService<Accepted> {
    fun addAccepted(accepted: Accepted): Int
    fun getAccepts(): ArrayList<Accepted>
    fun getMyAccepts(userId: String): ArrayList<Accepted>
    fun getTopicAccepts(topicId: Long): ArrayList<Accepted>
}
